Tests/kaas/k8s-default-storage-class/k8s-default-storage-class-check.py

```diff
@@ -0,0 +1,311 @@
+#!/usr/bin/env python3
+
+"""PersistentVolumeClaims checker
+
+Return codes:
+0:    Default StorageClass is available, and setup to SCS standard
+1:    Not able to connect to k8s api
+
+31:   Default storage class has no provisioner
+32:   None or more then one default Storage Class is defined
+
+41:   Not able to bind PersitantVolume to PersitantVolumeClaim
+42:   ReadWriteOnce is not a supported access mode
+
+All return codes between (and including) 1-19 as well as all return codes ending on 9
+can be seen as failures.
+
+Check given cloud for conformance with SCS standard regarding
+Default StorageClass and PersistentVolumeClaims, to be found under /Standards/scs-0211-v1-kaas-default-storage-class.md
+
+"""
+
+import getopt
+import sys
+import time
+import json
+import logging
+
+from kubernetes import client, config
+from helper import gen_sonobuoy_result_file
+
+import logging.config
+
+logger = logging.getLogger("k8s-default-storage-class-check")
+
+
+def setup_k8s_client(kubeconfigfile=None):
```

Comment:
As we will have several k8s conformity tests, I suggest to move this code to a extra module for re-use in other scripts.